currency_rate_update/__manifest__.py

```diff
@@ -3,29 +3,22 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
 {
-    'name': 'Currency Rate Update',
-    'version': '12.0.1.1.2',
-    'author':
-        'Camptocamp, '
-        'Brainbean Apps, '
-        'Odoo Community Association (OCA)',
-    'website': 'https://github.com/OCA/currency',
-    'license': 'AGPL-3',
-    'category': 'Financial Management/Configuration',
-    'summary': 'Update exchange rates using OCA modules',
-    'depends': [
-        'base',
-        'mail',
-        'account',
+    "name": "Currency Rate Update",
+    "version": "13.0.1.0.0",
+    "author": "Camptocamp, " "Brainbean Apps, " "Odoo Community Association (OCA)",
```

Comment:
The `" "` parts look really odd

Reply:
updated

Reply:
hm, does not show :(

Reply:
Seems autoformatting `" "` is still there